25/streams/developer-guide/write-streams.html

```diff
@@ -90,22 +90,22 @@
               <p class="last">See the section <a class="reference internal" href="datatypes.html#streams-developer-guide-serdes"><span class="std std-ref">Data Types and Serialization</span></a> for more information about Serializers/Deserializers.</p>
           </div>
           <p>Example <code class="docutils literal"><span class="pre">pom.xml</span></code> snippet when using Maven:</p>
-          <pre class="line-numbers"><code class="language-xml"><dependency>
-    <groupId>org.apache.kafka</groupId>
-    <artifactId>kafka-streams</artifactId>
-    <version>{{fullDotVersion}}</version>
-</dependency>
-<dependency>
-    <groupId>org.apache.kafka</groupId>
-    <artifactId>kafka-clients</artifactId>
-    <version>{{fullDotVersion}}</version>
-</dependency>
+          <pre class="line-numbers"><code class="language-xml">&lt;dependency>
+    &lt;groupId>org.apache.kafka&lt;/groupId>
```

Comment:
Should the `>` be replaced too?

Reply:
theoretically it should be generated with html entities for both gt and lt, but the opening is the only one that matters for highlighting the syntax. it's not an easy thing to find/replace so we'd have to do an audit of all the docs.